### DISCUSSION TITLE
In the page system status all reading are in English, could not be translated

### DISCUSSION THREAD

**Discussion by DiBochev** [NightscoutFoundation/xDrip#3259] [Bug reports]
In the page system status all reading are in English, could not be translated to any language.
another example: data source menu: G5/G6/G7/1 Transmitter, Libre Bluetooth, Libre wifi

**Comment by Navid200** [comment]
The system status page and the logs are used extensively by me and others like me who provide customer service in forums helping users bring up xDrip or address something that isn't working.

Translating those will make my job much harder.

What's on the system status page usually is simple words that hopefully are known by most even though they are in English.

Logs are similar.  But, more importantly, if you can give me an example of a log that is critical for fixing a problem that you think should be translated, I would appreciate it.
In a case like that, we may be able to do something to improve the situation.

Likewise, please give me an example of an item on the system status page that you believe should be translated.

Thanks

**Reply by DiBochev** [reply]
My current opinion is is that the application logs must be in English and there should be no discussion on this topics! 
But the discussion here is that many items from the menu is in English and there is no translation strings related to them. For other example:
Menu NovoPen: The title "NovoPen Insulin pen" could not be translated to any language. "OpenNov support" "NFC dose downloading of models 6 and Echo Plus" and etc...
Here are some screen shots from the application:
![411588903_723019139806771_4738785774883156643_n](https://github.com/NightscoutFoundation/xDrip/assets/7621587/ca7d8bbf-9681-4699-95c1-e540efc4af90)
![410300976_905801927770804_5353633891867125031_n](https://github.com/NightscoutFoundation/xDrip/assets/7621587/26c9789c-3eac-464f-9950-97daa61dc023)
![411984239_892681505834649_565365296116963290_n](https://github.com/NightscoutFoundation/xDrip/assets/7621587/622bb541-3484-41c3-9c80-8af990ccaaee)
![411647698_1761747374254169_1551496218849660435_n](https://github.com/NightscoutFoundation/xDrip/assets/7621587/97f99888-603a-4501-9a2c-137bf7cc336e)

![355178822_2495223897293246_8398930755203786267_n](https://github.com/NightscoutFoundation/xDrip/assets/7621587/25354001-d9cb-4692-9332-ad4f6b85cfb0)

**Comment by DiBochev** [comment]
should i open bug for the sings?

**Reply by Navid200** [reply]
No, please don't.
Developers know that every single word in xDrip is not translated.
As we go through the code, we add them.

There are also many submissions already that bring in more words into the translated category.
Opening a new issue is not going to change anything other than cluttering the issues platform.

The title of this discussion includes system status.
I explained why we don't translate words on system status.  You said you did not mean system status.  Yet, you have not corrected the title.

Discussions can be used to inform developers that users need something also.  But, it doesn't help if the discussions is not clear.
